schema-generic.graphql

```diff
@@ -6,16 +6,21 @@ enum Network {
   ARBITRUM_ONE
   AURORA
   AVALANCHE
+  BOBA
   BSC # aka BNB Chain
   CELO
+  CHAPEL
```

Comment:
Do you know what Chapel, Clover, and POA are? I've never heard of them. I thought xDAI used to be POA

Reply:
Chapel is BSC testnet - removed

Clover seems to be a polkadot parachain - https://parachains.info/details/clover_finance

and yeah i found out poa finished merging into gnosis on may 31 2022 and POA token is not supposed to have value anymore. It is now a decentralized testnet essentially